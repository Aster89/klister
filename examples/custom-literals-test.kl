#lang "custom-literals.kl"

(example 42)
(example (+ 2 2))
(example "foo")
