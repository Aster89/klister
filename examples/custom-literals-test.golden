43 : Integer
6 : Integer
"foo!" : String
